RedditSharp/Reddit.cs

```diff
@@ -581,6 +655,7 @@ public Comment GetComment(Uri uri)
         /// <param name="sortE">Order by <see cref="Sorting"/></param>
         /// <param name="timeE">Order by <see cref="TimeSorting"/></param>
         /// <returns></returns>
+        [Obsolete("time search was discontinued by reddit")]
```

Comment:
Damnit...

Reply:
more specifically elastic search was discontinued and timestamps weren't included in the new search